src/include/ipc/dai-intel.h

```diff
@@ -60,6 +60,16 @@
 /* DMIC max. four controllers for eight microphone channels */
 #define SOF_DAI_INTEL_DMIC_NUM_CTRL			4
 
+/* SSP clocks to enable/disable
+ *
+ * Macros for clk_id field in sof_ipc_dai_ssp_clkctrl struct.
+ */
+
+/* mclk */
+#define SOF_DAI_INTEL_SSP_CLKCTRL_MCLK			1
```

Comment:
lets keep it generic and remove "SSP" from these names.

Reply:
fixed